modules/ols-creating-lightspeed-custom-resource-file-using-cli.adoc

```diff
@@ -61,12 +58,12 @@ spec:
     providers:
       - credentialsSecretRef:
           name: credentials
-        deploymentName: <USE THE NAME OF THE DEPLOYMENT YOU CREATED EARLIER>
+        deploymentName: <Azure_AI_deployment_name>
```

Comment:
I believe that these should still be lowercase, even if it's the name of a product, i.e.
`<azure_ai_deployment_name>`

I can't see any examples in our style guides that give exceptions for product names, but this might be something we don't have a specific rule for, in which case it should maybe be raised with the larger OCP team to add a rule.